mne/cov.py

```diff
@@ -1740,7 +1747,7 @@ def _estimate_rank_meeg_cov(data, info, scalings, tol=1e-4,
         The rescaling method to be applied. If dict, it will override the
         following default dict:
 
-            dict(mag=1e15, grad=1e13, eeg=1e6)
+            dict(eeg=1e6, grad=1e13, mag=1e15)
```

Comment:
I'll revert, but just fyi, this is the order for the scalings dict in mne/viz/utils.py ...